modules/mobile/spec/support/helpers/sis_session_helper.rb

```diff
@@ -9,16 +9,31 @@ def sis_bearer_token
     @sis_bearer_token ||= SignIn::AccessTokenJwtEncoder.new(access_token: sis_access_token).perform
   end
 
-  def sis_user
-    @sis_user ||= create(:user, :api_auth, uuid: sis_access_token.user_uuid)
+  def sis_user(*args)
```

Comment:
This effectively enforces using the `api_auth` trait. All of our current specs require it. If we ever need a non-api_auth trait, it would be fairly easy to make that work without breaking everything else.